packages/cloud-cognitive/src/components/AddSelect/AddSelectFormControl.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {
+  RadioButton,
+  RadioButtonChecked,
+  Checkbox,
+  CheckboxChecked,
+} from '@carbon/react/icons';
+import PropTypes from 'prop-types';
+import { pkg } from '../../settings';
+import { UserProfileImage } from '../UserProfileImage';
```

Comment:
Wonder if we should consider having consumers pass in their own `UserProfileImage` later since this will eventually get deprecated and replaced with `UserAvatar` in v2 ... we might need to support both types? Or we could just update to `UserAvatar` when that time comes. 🤔